app/src/main/java/com/goldenraven/padawanwallet/Wallet.kt

```diff
@@ -0,0 +1,141 @@
+package com.goldenraven.padawanwallet
+
+import android.content.SharedPreferences
+import org.bitcoindevkit.bdkjni.Lib
+import org.bitcoindevkit.bdkjni.Types.*
+import timber.log.Timber
+
+object Wallet {
+
+    private var lib: Lib
+    private lateinit var walletPtr: WalletPtr
+    private var name: String = "Padawan Testnet 0"
+    private var network: String = "testnet"
+    private lateinit var path: String
+    private lateinit var descriptor: String
+    private lateinit var changeDescriptor: String
+    private var electrumURL: String = "tcp://testnet.aranguren.org:51001"
+
+    init {
+        // load bitcoindevkit
+        Lib.load()
+        this.lib = Lib()
+        // setDefaults()
+    }
+
+    public fun helloFrom(location: String): Unit {
+        Timber.i("[PADAWANLOGS] New wallet is alive and well, reporting from $location")
+    }
+
+    public fun setPath(path: String) {
```

Comment:
The method might not need such a comment if it took an application context instead, and then was responsible for getting the correct path on its own